src/igvfd/schemas/construct_library_set.json

```diff
@@ -132,59 +138,161 @@
+                                            },
+                                            {
+                                                "const": "genes"
+                                            }
+                                        ]
+                                    }
                                 },
-                                {
-                                    "required": [
-                                        "upper_bound_guide_coverage"
+                                "not": {
+                                    "anyOf": [
+                                        {
+                                            "required": [
+                                                "guide_type"
+                                            ]
+                                        },
+                                        {
+                                            "required": [
+                                                "tiling_modality"
+                                            ]
+                                        },
+                                        {
+                                            "required": [
+                                                "average_guide_coverage"
+                                            ]
+                                        },
+                                        {
+                                            "required": [
+                                                "lower_bound_guide_coverage"
+                                            ]
+                                        },
+                                        {
+                                            "required": [
+                                                "upper_bound_guide_coverage"
+                                            ]
+                                        }
                                     ]
                                 }
-                            ]
-                        }
+                            }
+                        ]
                     }
                 }
             ]
         },
         "scope": {
-            "comment": "If scope is genes then genes is required, if scope is tile then both tile and genes are required, if scope is exon then both genes and exon are required, if scope is genome-wide then tile, exon, genes, and loci shouldn't be specified.",
+            "comment": "If scope is genes then genes, small_scale_gene_list, or large_scale_gene_list is required, if scope is tile then both tile and genes, small_scale_gene_list, or large_scale_gene_list are required, if scope is loci then loci, small_scale_loci_list, or large_scale_loci_list are required, if scope is exon then both genes or small_scale_gene_list and exon are required, if scope is genome-wide then tile, exon, small_scale_gene_list, large_scale_gene_list, loci, small_scale_loci_list, and large_scale_loci_list shouldn't be specified.",
```

Comment:
large scale list of genes can not be relevant for exon/tile

Reply:
fixed